src/Compilers/Core/Portable/Symbols/Nullability.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis
+{
+    // PROTOTYPE(nullable-api): Doc comment
+    public enum Nullability
+    {
+        NotApplicable,
```

Comment:
When `GetTypeInfo` is called on a statement, rather than an expression.

---
In reply to: [252430661](https://github.com/dotnet/roslyn/pull/32899#discussion_r252430661) [](ancestors = 252430661)